recipes/libxc/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "libxc" %}
+{% set version = "4.3.3" %}
+{% set sha256 = "4dedcadbb5755b6da1836effabf0ad0d77a1b6fb6760d847a456d1ace9145faa" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  #url: http://www.tddft.org/programs/octopus/down.php?file=libxc/4.3.3/libxc-4.3.3.tar.gz
```

Comment:
These comments should be deleted I think?

Reply:
Yes, you're right.